rotkehlchen/accounting/constants.py

```diff
@@ -126,6 +126,19 @@
     HistoryEventType.BURN: {
         HistoryEventSubType.NFT: {DEFAULT: EventCategory.BURN_NFT},
     },
+    HistoryEventType.MULTI: {
+        HistoryEventSubType.SPEND: {DEFAULT: EventCategory.SWAP_OUT},
+        HistoryEventSubType.RECEIVE: {DEFAULT: EventCategory.SWAP_IN},
+        HistoryEventSubType.FEE: {DEFAULT: EventCategory.FEE},
+    },
+}
+
+EVENT_GROUPING_ORDER = {
```

Comment:
this will need to change I guess with the type changes but also explain in a short comment here where this is used